test/test_citusdb_baseRIwPEmodule.yml

```diff
@@ -0,0 +1,259 @@
+---
+- hosts: citus_coordinator
+  vars:
+    pp_superadmin: "postgres"
+    pp_superpassword: "password"
+
+  tasks:
+    - name: Install requirements for postgresql Ansible tasks
+      yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - python-psycopg2
+
+    - name: Download users data (31.7MB)
+#     shell: wget https://examples.citusdata.com/tutorial/users.csv -O /tmp/users.csv 
+      get_url:
+        url: https://examples.citusdata.com/tutorial/users.csv
+        dest: /tmp/users.csv
+        mode: 0755
+
+    - name: Download events data (74.7MB)
+#     shell: wget https://examples.citusdata.com/tutorial/events.csv -O /tmp/events.csv 
+      get_url:
+        url: https://examples.citusdata.com/tutorial/events.csv
+        dest: /tmp/events.csv
+        mode: 0755
+
+    - name: Ensure that a connection can be made
+      postgresql_exec:
+        login_user: "{{ pp_dba_username }}"
+        login_password: "{{ pp_dba_password }}"
+        login_host: citus_coordinator
+        #login_host: "{{ hostvars[groups['db_server'][0]]['ansible_ssh_host'] }}"
+        port: "{{ pp_serverport }}"
+        db: "postgres"
+        script: "SELECT 1 AS TEST;"
+      register: sql_out
+
+    - name: print sql results
+      debug:
+        msg: "{{ sql_out }}"
+        #msg="-----------------------------------------------" 
+      #with_nested: "{{ query_result.stdout }}"
+
+    - name: create table users
+      postgresql_exec:
+        login_user: "{{ pp_superadmin }}"
+        login_password: "{{ pp_superpassword }}"
+        login_host: citus_coordinator
+        port: "{{ pp_serverport }}"
+        db: "postgres"
+        script: "create table IF NOT EXISTS github_users(user_id bigint,url text,login text,avatar_url text,
+          gravatar_id text,display_login text);"
+      register:  create_table_users_sql_results
+ 
+
+    - name: print sql results
+      debug:
+        msg: "{{ create_table_users_sql_results['status'] }}"
+        #msg="-----------------------------------------------" 
+      #with_nested: "{{ query_result.stdout }}"
+ 
+#   - name: Assert results are correct
+#     assert:
+#       that: 
+#         - "{{ create_table_users_sql_results['status'] }} s defined"
```

Comment:
@lel99999 I would like to try to use `assert` if possible, for each test, along with the `debug`'s, since the goal is to have the Jenkins deployment job fail if these tests don't pass.